lab_1_classify_by_unigrams/main.py

```diff
@@ -34,6 +43,14 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(tokens, list) or len(tokens) == 0 or None in tokens:
+        return None
+
+    freq_dict = {}
+    for character in tokens:
+        freq_dict[character] = (tokens.count(character) / len(tokens))
```

Comment:
You call `list.count`  for each element from the list. What if there are some identical elements and you get the same results for this elements but could run it only for unique letters. Can you optimize it?

Reply:
done